woodwork/table_accessor.py

```diff
@@ -778,9 +779,8 @@ def rename(self, columns, inplace=False):
     def mutual_information_dict(self, num_bins=10, nrows=None, include_index=False, callback=None):
         """
         Calculates mutual information between all pairs of columns in the DataFrame that
-        support mutual information. Logical Types that support mutual information are as
-        follows:  Age, AgeFractional, AgeNullable, Boolean, BooleanNullable, Categorical, CountryCode, Datetime, Double,
-        Integer, IntegerNullable, Ordinal, PostalCode, and SubRegionCode
+        support mutual information. Use get_valid_mi_types to see which Logical Types support
```

Comment:
I think you can put in docstrings with code like this. You can actually run them as tests

```suggestion
        support mutual information. Use get_valid_mi_types to see which Logical Types support
        >>> from woodwork.utils import get_valid_mi_types
        >>> get_valid_mi_types()
        [Age, AgeFractional, AgeNullable, Boolean, BooleanNullable, 
         Categorical, CountryCode,Datetime, Double, Integer, IntegerNullable, 
         Ordinal, PostalCode, SubRegionCode]
```

- You will need to make sure setup.cfg has this
```
addopts = --ignore "boilerplates/" --doctest-modules
```

- We do this in premium primitives
- https://github.com/alteryx/premium-primitives/blob/main/setup.cfg#L4
- https://github.com/alteryx/premium-primitives/blob/main/premium_primitives/age_over_18.py#L22